wallet-new/server/Cardano/Wallet/API/V1/Swagger.hs

```diff
@@ -345,42 +363,52 @@ Since you have, for now, only a single wallet, you'll see something like this:
 }
 ```
 
-Each account has at least one address, all listed under the `addresses` field. You can communicate one of these addresses to receive money on the associated account.
+All the wallet's accounts are listed under the `addresses` field. You can communicate one of
+these addresses to receive _Ada_ on the associated account.
 
 
-Sending Money
--------------
+Sending Ada
+-----------
 
-In order to send money from one of your account to another address, you can create a new payment transaction using the `POST /api/v1/transactions` endpoint as follow:
+In order to send _Ada_ from one of your accounts to another address, you must create a new
+payment transaction using the [`POST /api/v1/transactions`](#tag/Transactions%2Fpaths%2F~1api~1v1~1transactions%2Fpost)
+endpoint as follows:
 
 ```
 curl -X POST https://localhost:8090/api/v1/transactions \
-     -H "Content-Type: application/json; charset=utf-8" \
-     -H "Accept: application/json; charset=utf-8"       \
-     --cacert ./scripts/tls-files/ca.crt                \
-     -d '{                                              \
-  "destinations": [{                                    \
-    "amount": 14,                                       \
-    "address": "A7k5bz1QR2...Tx561NNmfF"                \
-  }],                                                   \
-  "source": {                                           \
-    "accountIndex": 0,                                  \
-    "walletId": "Ae2tdPwUPE...8V3AVTnqGZ"               \
-  }                                                     \
+  -H "Accept: application/json; charset=utf-8" \
+  -H "Content-Type: application/json; charset=utf-8" \
+  --cacert ./scripts/tls-files/ca.crt \
+  -d '{
+  "destinations": [{
+    "amount": 14,
+    "address": "A7k5bz1QR2...Tx561NNmfF"
+  }],
+  "source": {
+    "accountIndex": 0,
+    "walletId": "Ae2tdPwUPE...8V3AVTnqGZ"
+  },
+  "spendingPassword": "5416b2988745725998907addf4613c9b0764f04959030e1b81c603b920a115d0"
 }'
 ```
 
-Note that, in order to perform a transaction, you need to have some existing coins on the source account! Beside, the Cardano API is designed to accomodate multiple recipients payments out-of-the-box; notice how `destinations` is a list of addresses.
+Note that, in order to perform a transaction, you need to have enough existing coins on the
```

Comment:
you used `coins` here -- let's stick with ADA 😉

Reply:
Ah!